### PR TITLE
Error server early if the file is not found or not YAML/JSON

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

File: lib/validate.js [hunk 1]
```diff
@@ -1223,7 +1223,4 @@ function validate(openapi, options, callback) {
     });
 }
 
-module.exports = {
-    validateSync: validateSync,
-    validate: validate
-}
+module.exports = { validateSync, validate }
```
Comment (Whitespace) [inline lib/validate.js]: Why is this a one-liner instead of multiline like other files?

File: lint.js [hunk 2]
```diff
@@ -88,6 +88,7 @@ const main = (str, callback) => {
     .catch(function(err) {
         options.status = 'rejected';
         console.warn(err);
+        process.exitCode = 0;
```
Comment (Whitespace) [inline lint.js]: Unsure why this would consume other `errorCode`s

File: lib/validate.js (hunk 1 above)
Comment (philsturgeon) [reply lib/validate.js]: I was gonna make them all one line, musta missed. It's short enough to be a one liner right?

File: lint.js (hunk 2 above)
Comment (philsturgeon) [reply lint.js]: Switched to `process.exit(0)` but its the same thing in this instance.

**Pull request merged**